test/polydb.jl

```diff
@@ -0,0 +1,163 @@
+using Mongoc
+
+@testset "polyDB" begin
+    add_constraints_poly = get(ENV, "POLYDB_SERVER_URI", "") == "" ? ["DIM" => Dict("\$lte" => 3)] : []
```

Comment:
what is `lte`?

Reply:
mongo syntax for `less than or equal`, the constraint with which the database dump was created